lib/mocha.js

```diff
@@ -244,14 +244,22 @@ Mocha.prototype._growl = function (runner, reporter) {
 
   runner.on('end', function () {
     var stats = reporter.stats;
+    var errMsg = 'The prerequisites for Growl support are missing. Please see https://github.com/mochajs/mocha/wiki/Growl-Notifications for details';
+    var msg = '';
+
     if (stats.failures) {
-      var msg = stats.failures + ' of ' + runner.total + ' tests failed';
-      notify(msg, { name: 'mocha', title: 'Failed', image: image('error') });
+      msg = stats.failures + ' of ' + runner.total + ' tests failed';
+      notify(msg, { name: 'mocha', title: 'Failed', image: image('error') }, function (err) {
```

Comment:
maybe we can abstract the error into a function `onNotifyError` or such then use it in both places. This way the code is better refactored and easier to understand

Reply:
Definitely, DRY is better